mentoring/README.md

```diff
@@ -5,57 +5,53 @@ description: |
-, it's necessary to keep a continuous pipeline of quality contributors and we want you to hang around!
+and organization of every kind. For Kubernetes, the larger the project becomes,
+it's necessary to keep a continuous pipeline of quality contributors and we want you to hang around!
 
-## Current mentoring activities:
+## Resources
+- [New Contributors Guide](https://docs.google.com/presentation/d/1WO7vj33cVy1-rOav3CevAGrw97VrbIAFo_SQXbUwy28/edit#slide=id.g2f2865a3246_0_63)
+- [Mentor Guides](/mentoring/mentors/mentor-guide.md)
+- [Mentee Guides](/mentoring/mentees/group-mentee-guide.md)
+- [SIG Office Hours](/mentoring/resources/office-hours.md)
 
-Please reach out to #sig-contribex on slack or come to a mentoring meeting to get involved in planning //TODO add contribex README when this is updated
-
-SIG's office hours / mentoring
-
-- [Office hours / mentoring](/mentoring/programs/office-hours.md)
-
-Long Term Contributor Ladder Growth
-
-- Through [Group Mentoring Cohorts](/mentoring/programs/group-mentoring.md)
-
-Specific topics and activities
+## Programs
 
-- [The 1:1 Hour](/mentoring/programs/the1-on-1hour.md)
+| Program                                                                                         | Status                   | Target Date                                                                   |
+|-------------------------------------------------------------------------------------------------|--------------------------|-------------------------------------------------------------------------------|
+| [Group Mentoring Cohorts](/mentoring/programs/2024/groupmentorship/group-mentor-planning.md) | **Planning in progress** | [Sept-Nov](/mentoring/programs/2024/groupmentorship/group-mentor-planning.md) |
 
-Role based shadow programs
 
-- [List of programs](/mentoring/programs/shadow-roles.md)
+### Self Paced
+- [Contributor Workshop](/mentoring/programs/contributor-workshop/README.md)
+- [Videos of New Contributor Workshop](https://www.youtube.com/playlist?list=PL69nYSiGNLP3M5X7stuD7N4r3uP2PZQUx)
 
-Students
+### Role based shadow programs
 
-- [Google Summer of Code](/mentoring/programs/google-summer-of-code.md)
+- [List of programs](/mentoring/programs/2024/shadow-roles.md)
 
-1:1 full-time mentoring
+### Mentoring Events
 
-- [LFX Mentorship](/mentoring/programs/lfx-mentorship.md)
 
-Tech Writers
+- Pod Mentoring! [aka group mentoring](/mentoring/programs/archive/mentoring-events.md)
 
-- [Google Season of Docs](/mentoring/programs/google-season-of-docs.md)
+### External programs
+- [LFX Mentorship](/mentoring/processes/lfx-mentorship.md)
 
-Groups Traditionally Underrepresented in Tech
+## Archive
+- [Mentoring Programs](/mentoring/programs/archive/README.md)
 
-- [Outreachy](/mentoring/programs/outreachy.md)
+Please reach out to #sig-contribex on slack or come to a mentoring meeting to get involved in planning //TODO add contribex README when this is updated
```

Comment:
Are we planning to re-start the Mentoring Subproject Meetings?

Reply:
That was actually on the original page - I just moved it. Although in saying that, and after speaking to Paris, it does seem we will need some meetings.

Reply:
Basically, I've updated the landing page to make it "current", and restructured the folder structure. I'm going to add screen shots or push my changes to a personal space so it can be reviewed

Reply:
Before and Proposed (for review) landing pages....
![LandingPage-Current](https://github.com/user-attachments/assets/e49aa382-9310-4637-bb6b-d980090c54f4)
![LandingPage-Proposed](https://github.com/user-attachments/assets/520ce906-4ac3-4bef-a740-d40d1f4856e5)

Basically removed anything that isn't current, added a plan (which very much needs discussion), and made some other updates for the new contribution guide.

Reply:
Folder Structure.
![communityMentoring-CurrentFolderLayout](https://github.com/user-attachments/assets/c002e8cd-e2dc-496e-819c-63b225d17e0c)
![communityMentoring-ProposedFolderLayout](https://github.com/user-attachments/assets/2de6bf62-9054-4a99-8d15-18132780eba0)

Here, I just tried to organize it (and archive the "old" pages")

Reply:
Also  added a new plan which I want to discuss:

![groupMentorPlanning](https://github.com/user-attachments/assets/5f344332-35aa-4fcc-8561-6a09da037e64)

- one change to the  contributor-site (Land page)
- several changes to the community/mentoring github folder structure and one new page (planning)